test/Microsoft.AspNetCore.Razor.Language.Test/IntegrationTests/TestTagHelperDescriptors.cs

```diff
@@ -158,7 +162,7 @@ public static IEnumerable<TagHelperDescriptor> EnumTagHelperDescriptors
                                 .Name("catch-all")
                                 .PropertyName("CatchAll")
                                 .AsEnum()
-                                .TypeName(typeof(MyEnum).FullName),
+                                .TypeName(GetFullName(typeof(MyEnum))),
```

Comment:
Can we just hardcode the actual string you expect to see? This is too much magic for the test. You copied product code into here to do this.